gulpfile.js

```diff
@@ -256,7 +296,7 @@ function combine() {
     });
 }
 
-gulp.task('combine', gulp.series('generateStubs', combine));
+gulp.task('combine', gulp.series('build', combine));
```

Comment:
I never really used `generateStubs`, which looks to be gone now. Is this the WebGL-stubbed version of the tests? Isn't this what travis uses when running the tests?

Reply:
No, that's `getWebGLStub`, which is still very much around.  `generateStubs` is what made testing combined/minified CesiumJS possible.  They were requirejs compatible shims that called build version under the hood and were injected into the tests if release/combined was requested.  It was just completely under the hood so you didn't know you were using them, you never had a reason to call it directly.